axelrod/strategies/titfortat.py

```diff
@@ -91,7 +91,36 @@ class TwoTitsForTat(Player):
     @staticmethod
     def strategy(opponent: Player) -> Action:
         return D if D in opponent.history[-2:] else C
+    
+class DynamicTitForTat(Player):
+    """A player starts by cooperating and then mimics previous move by
```

Comment:
If you have found this strategy elsewhere we'll need to reference the source, if not we still need to say it's an original strategy. There's information about what we need in the docstring here: http://axelrod.readthedocs.io/en/stable/tutorials/contributing/strategy/docstrings.html

On that note, I'm not entirely sure this is a good name for the strategy, if it's called this somewhere else then we will stick with it though.